ui/org.eclipse.pde.genericeditor.extension.tests/META-INF/MANIFEST.MF

```diff
@@ -13,6 +13,7 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)",
  org.eclipse.pde.genericeditor.extension,
  org.eclipse.pde.core,
  org.eclipse.equinox.p2.metadata,
+ org.eclipse.equinox.p2.transport.ecf,
```

Comment:
Requiring a concrete _implementation_ her is likely wrong in many ways.

Instead the provider needs to provide approciate capbilty (service or implementor) and the consumer (where NPE occures) require appropriate requirement (either service or implementor capabilty)


A consumer bunlde like this should not need any of those!

Reply:
The fact that all the tests pass seems like a good thing.  Future improvement could be saved for the future...